### PR TITLE
 Museum: Display hydrated items for items taken outside of the museum

### PR DESCRIPTION
**Pull request by romangraef**

### CODE REVIEW

**Comment by NopoTheGamer** [comment]
image
image
kills fps :(

**Review by Lulonaut — DISMISSED** [review]

**Comment by jani270** [comment]
me when not tested ingame

**Comment by Lulonaut** [comment]
I don't have minecraft on windows 😭

**Comment by jani270** [comment]
> I don't have minecraft on windows 😭

😭

**Review by NopoTheGamer — APPROVED** [review]

**Review by hannibal002 — CHANGES_REQUESTED** [review]
Please disable the feature by default. Additionally, I don't like the highlighting at all. Maybe just add an extra line in the lore?

**Comment by hannibal002** [comment]
Furthermore, the config does not save after reloading. Can reproduce but can't find the error in code.

https://discord.com/channels/516977525906341928/1055881432465166476/1055881432465166476

**Comment by Lulonaut** [comment]
> Furthermore, the config does not save after reloading. Can reproduce but can't find the error in code.
> 
> https://discord.com/channels/516977525906341928/1055881432465166476/1055881432465166476

the Expose annotation is missing at the config values

**Comment by romangraef** [comment]
> Please disable the feature by default. Additionally, I don't like the highlighting at all. Maybe just add an extra line in the lore?
You can disable the highlighting already and there is already an extra line in the lore.

**Review by NopoTheGamer — APPROVED** [review]
I would prefer it if you didnt force push all of your changes

**Comment by romangraef** [comment]
I would prefer to have clean commits. You can compare two force pushes using github by clicking [COMPARE] next to the force push message (or on your client use standard git diff commands)
image

**Comment by NopoTheGamer** [comment]
1. clean commits dont do anything? it doesn't effect the main repo 2. comparing isn't that helpful when you also merged master in the same push

**Review by Lulonaut — DISMISSED** [review]

**Review by NopoTheGamer — CHANGES_REQUESTED** [review]
i was thiking especially with lulo adding something for museum, why don't we make a config section for museum stuff. misc is getting kinda full

**Comment by jani270** [comment]
Im fairly sure this is caused by other funny mods: https://i.imgur.com/T6o5toV.gif (Also the gif is like 15 fps so its alot faster ingame)
Just want to bring that up because im sure it will come up in support

**Review by Lulonaut — APPROVED** [review]

**Review by romangraef — COMMENTED** [review]
Just noticed that this does not properly displace the inventory buttons on the right side. Maybe we should add a InventoryButtonPositionEvent so that we dont put more shit into that weird if else in RenderListener

**Comment by hannibal002** [comment]
When will the PR remove the cheap museum overlay stuff? The current version here still contains bugs.

File: src/main/java/io/github/moulberry/notenoughupdates/options/NEUConfig.java [hunk 1]
```diff
@@ -181,6 +182,12 @@ public void executeRunnable(int runnableId) {
 	)
 	public Misc misc = new Misc();
 
+	@Expose
+	@Category(
+		name = "Museum",desc = "Musem overlays"
+	)
+	public Museum museum = new Museum();
```
Comment (NopoTheGamer) [inline src/main/java/io/github/moulberry/notenoughupdates/options/NEUConfig.java:189]: I feel like this should be lower down

File: src/main/java/io/github/moulberry/notenoughupdates/options/NEUConfig.java [hunk 2]
```diff
@@ -246,8 +253,8 @@ public void executeRunnable(int runnableId) {
 
 	@Expose
 	@Category(
-		name = "Todo Overlay",
-		desc = "Todo Overlay"
+		name = "Misc Overlays",
```
Comment (NopoTheGamer) [inline src/main/java/io/github/moulberry/notenoughupdates/options/NEUConfig.java:256]: This change should be reverted

**Review by NopoTheGamer — CHANGES_REQUESTED** [review]

File: src/main/java/io/github/moulberry/notenoughupdates/options/NEUConfig.java (hunk 1 above)
Comment (jani270) [reply src/main/java/io/github/moulberry/notenoughupdates/options/NEUConfig.java:189]: Also "Musem overlays"

**Review by NopoTheGamer — APPROVED** [review]

**Review by Lulonaut — APPROVED** [review]

**Comment by romangraef** [comment]
Discontinued in favor or #621

**Pull request closed without merging**